app/main.py

```diff
@@ -1,3 +1,76 @@
+from __future__ import annotations
+
+
 class Distance:
-    # Write your code here
-    pass
+    def __init__(self, km: float | int) -> None:
+        self.km = float(km)
```

Comment:
Do not convert to float value here
```suggestion
        self.km = km
```